KuduSync.NET/Program.cs

```diff
@@ -7,20 +7,53 @@ namespace KuduSync.NET
 {
     class Program
     {
+        public const string AppOfflineFileName = "app_offline.htm";
+        private const string AppOfflineFileContent =
+            "<!DOCTYPE html PUBLIC \"-//W3C//DTD XHTML 1.0 Transitional//EN\" \"http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd\">" +
+            "<html xmlns=\"http://www.w3.org/1999/xhtml\" > " +
+            "<head> " +
+            "    <title>Site Under Construction</title> " +
+            "</head> " +
+            "<body>" +
+            "<!--                                                                                                                        " +
+            "                                                                                                                            " +
+            "    Adding additional hidden content so that IE Friendly Errors don't prevent                                               " +
+            "    this message from displaying (note: it will show a 'friendly' 404                                                       " +
+            "    error if the content isn't of a certain size).                                                                          " +
+            "                                                                                                                            " +
+            "-->" +
+            "</body>" +
+            "</html>";
+        private const string AppOfflineSetting = "KUDU_APP_OFFLINE_CREATION";
```

Comment:
We normally use an SCM_ prefix for Kudu things. e.g. see [here](https://github.com/projectkudu/kudu/wiki/Configurable-settings#repository-and-deployment-related-settings)

Reply:
Also, I think the name should be more action oriented, e.g. SCM_CREATE_APP_OFFLINE